### PR TITLE
Bluesky integration

### PR DESCRIPTION
**Pull request by hermanschutte**
This plugin allows Auto-GPT to retrieve posts from Bluesky and create new posts. With Twitter becoming much stricter with developer apps, Bluesky is a good alternative.

### CODE REVIEW

**Comment by p-i-** [comment]
This is a mass message from the AutoGPT core team to all open Plugin-PRs

We're in the process of re-architecting:
  https://github.com/Significant-Gravitas/Auto-GPT/wiki/Architecting

Please be patient as we work to clean up the AutoGPT core.

If you already have an existing plug-in that’s affected, please message @ntindle on Discord so we can get your plugin spinning on the new 
architecture.

If you're working on a plugin, please bear in mind that one of our primary goals in re-architecting is to make life MUCH easier for you. 
We've taken aboard feedback from plugin developers. We're on it, so please give us some time!

<3 The AutoGPT Team

**Review by ntindle — CHANGES_REQUESTED** [review]
Add atprototools to the requirements.txt

**Review by ntindle — CHANGES_REQUESTED** [review]
`SYSTEM:  Command post_to_bluesky returned: Error: 'Session' object has no attribute 'post_bloot'`

**Comment by ntindle** [comment]
Consider https://github.com/marshalX/atproto over the library chosen as the library chosen breaks Python conventions.

**Comment by ntindle** [comment]
It looks like one of your tests failed. Can you take a look?

**Comment by ntindle** [comment]
cce90813-dabf-4051-a70d-5ae2702954b4
This is the image it generated to post btw

**Comment by ntindle** [comment]
Would you take a look at the CI failure? 

https://github.com/Significant-Gravitas/Auto-GPT-Plugins/actions/runs/4936955336/jobs/8830687797?pr=97

**Comment by hermanschutte** [comment]
> Would you take a look at the CI failure?
> 
> https://github.com/Significant-Gravitas/Auto-GPT-Plugins/actions/runs/4936955336/jobs/8830687797?pr=97

I'm getting this error when running the tests:

```
_ ERROR collecting src/autogpt_plugins/bluesky/bluesky_plugin/test_bluesky_plugin.py _
ImportError while importing test module '/home/runner/work/Auto-GPT-Plugins/Auto-GPT-Plugins/src/autogpt_plugins/bluesky/bluesky_plugin/test_bluesky_plugin.py'.
Hint: make sure your test modules/packages have valid Python names.
Traceback:
/opt/hostedtoolcache/Python/3.10.11/x64/lib/python3.10/importlib/__init__.py:126: in import_module
    return _bootstrap._gcd_import(name[level:], package, level)
src/autogpt_plugins/bluesky/bluesky_plugin/test_bluesky_plugin.py:2: in <module>
    from .bluesky_plugin import (
E   ImportError: attempted relative import with no known parent package
```

Hi. No idea why it would be complaining about the module names? It matches the format of the other plugins as far as I can tell. Any ideas?

File: src/autogpt_plugins/bluesky/bluesky_plugin/test_bluesky_plugin.py [hunk 1]
```diff
@@ -0,0 +1,33 @@
...
+        os.environ.pop("BLUESKY_APP_PASSWORD", None)
+
+    def test_username_and_pwd_set(self):
+        self.assertTrue(username_and_pwd_set())
+
+    def test_post_message(self):
+        self.assertIsInstance(post_message(MOCK_MESSAGE), str)
+
+    def test_get_latest_posts(self):
+        self.assertIsInstance(get_latest_posts(MOCK_USERNAME, 5), str)
```
Comment (ntindle) [inline src/autogpt_plugins/bluesky/bluesky_plugin/test_bluesky_plugin.py:29]: Can you swap to pytest?

**Review by ntindle — CHANGES_REQUESTED** [review]

File: src/autogpt_plugins/bluesky/bluesky_plugin/test_bluesky_plugin.py (hunk 1 above)
Comment (ntindle) [reply src/autogpt_plugins/bluesky/bluesky_plugin/test_bluesky_plugin.py:29]: jk thats for the other repo

**Pull request merged**